gbus/builder/builder.go

```diff
@@ -110,7 +112,8 @@ func (builder *defaultBuilder) Build(svcName string) gbus.Bus {
 	if builder.purgeOnStartup {
 		err := sagaStore.Purge()
 		if err != nil {
-			panic(err)
+			logerr := fmt.Errorf("grabbit faild on sagaStore.Purge. builder.txConnStr: %v,builder.connStr: %v, error: %v", builder.txConnStr, builder.connStr, err)
```

Comment:
same as the above

Reply:
why is this actin done twice? `sagaStore.Purge()` 
it is already happening inside the mysql block.,

Reply:
because it's crappy code that managed to slip away :)